### ISSUE TITLE
Login with Office 365 / Azure AD using SAML or Open ID Connect (OIDC)

### ISSUE THREAD

**Issue by lcorneliussen** [AnomalyInnovations/serverless-stack-com#430]
First of all thanks for the fabulous work beeing done on serverless-stack.com!

I've been integrating the Notes-App with Office 365 Logins and was wondering, if a chapter on how to achieve that would be welcome.

After I while I got it working both with Cognito SAML and OIDC Identity providers - so a chapter on it would in a way also cover how to integrate any other SAML or OICD-based authentication.

I used chapter discussions to document my findings so far:
https://discourse.serverless-stack.com/t/integrating-office-365-logon/1566

Any thoughts?

**Comment by jayair** [comment]
Yeah for sure! That would be a useful addition. Thanks for the forum post. If you are interested in putting together a draft for the chapter that would be awesome.
